lib/llm/providers/anthropic.rb

```diff
@@ -13,8 +13,8 @@ class Anthropic < Provider
 
     ##
     # @param secret (see LLM::Provider#initialize)
-    def initialize(secret)
-      super(secret, HOST)
+    def initialize(secret, host: HOST)
```

Comment:
I think we want something like this - right ?

```ruby
def initialize(secret, **options)
  super(secret, **{host: HOST}.merge(options))
end
```

Reply:
OH! true, what a blindspot